app/routes/groups_.$groupId.tsx

```diff
@@ -1,21 +1,69 @@
-import type { LoaderFunction } from '@remix-run/node';
+import type { Event } from '@prisma/client';
+import type { LoaderFunction, LoaderFunctionArgs, SerializeFrom } from '@remix-run/node';
 import { useParams, useLoaderData, Link } from '@remix-run/react';
 import { db } from '~/modules/database/db.server';
+import { EventCard } from '~/modules/events/card';
 import { json } from '@remix-run/node';
 
-export const loader: LoaderFunction = async ({ params }) => {
-  const group = await db.group.findFirstOrThrow({ where: { id: params.groupId } });
+export const loader: LoaderFunction = async ({ params }: LoaderFunctionArgs) => {
+  const group = await db.group.findFirstOrThrow({ where: { id: params.groupId }, include: { events: true } });
   return json({ group });
 };
 
+export function useDeserializeGroup() {
```

Comment:
trying to make this into a util function, but seems trickier than I thought

Reply:
`useLoaderData<typeof loader>();` alone doesn't work?

Reply:
no, with `useLoaderData<typeof loader>();` eslint won't complain, but the datatime would be a string which will cause some typing issue down the line.